### DISCUSSION TITLE
DALY battery voltage is displayed incorrectly

### DISCUSSION THREAD

**Discussion by cyborgxxl** [Louisvdw/dbus-serialbattery#82] [Q&A] [answered]
Does anyone have a clue or an idea why the measured battery voltage between Victron and DALY BMS is so extremely different in my installation? Does anyone else here also have this problem?

![Voltage](https://user-images.githubusercontent.com/6112343/149475555-b04a33ca-384d-431c-83ca-67a524bb80b6.jpg)

![photo_2022-01-14_09-25-42](https://user-images.githubusercontent.com/6112343/149475628-1081ff0f-b5b5-4c65-8adf-99abfd84612c.jpg)
![photo_2022-01-14_09-25-46](https://user-images.githubusercontent.com/6112343/149475629-7e7057af-3601-40c2-bf31-6bf46480fb58.jpg)

**Comment by Louisvdw** [comment, marked as answer]
It is normal that you will see small voltage drops in different parts of your system. 
It all depends on where that measurement is made and what the resistance and distance is of the cables between those points. Thinner wires will also have greater resistance. You will see this in the specs of your wire. If you look at Victron's PV calculator sheet for your MPPT you will see that the cable distance and thinkness is also part of the calculation. 

Another influence could also be how well your measurement shunt is calibrated.
